test/dlc_tests/conftest.py

```diff
@@ -124,8 +125,9 @@ def feature_aws_framework_present():
 
 def pytest_addoption(parser):
     default_images = test_utils.get_dlc_images()
+    images = default_images.split(" ") if default_images else []
```

Comment:
I encountered the case where default_images was None while testing. I just think we should check regardless, since the get_dlc_images does not guarantee not having a NoneType.